easyconfigs/o/ORCA/ORCA-5.0.4-foss-xtb-6.5.1-2022a.eb

```
@@ -0,0 +1,40 @@
+name = 'ORCA'
+version = '5.0.4'
+
```

```suggestion
versionsuffix = '-xtb-6.5.1'

```

and filename should be `ORCA-5.0.4-foss-2022a-xtb-6.5.1.eb`

---

f2c65b6